lib/iris/tests/unit/analysis/maths/test_divide.py

```diff
@@ -42,10 +39,7 @@ def cube_func(self):
 class TestMasking(tests.IrisTest_nometa, CubeArithmeticMaskingTestMixin):
     @property
     def data_op(self):
-        try:
-            return operator.div
-        except AttributeError:
-            return operator.truediv
+        return operator.truediv
```

Comment:
As the [above comment](https://github.com/SciTools/iris/pull/3507/files#r342056420).